extensions/import/src/test/wizard/pages/fileConfigPage.test.ts

```diff
@@ -41,26 +61,219 @@ describe('import extension wizard pages', function () {
 				]
 			],
 			columnInfo: undefined
-		}
+		};
+
+		// setting the default schema for the current connection. This schema should be the first value in the dropdown array
+		mockImportModel.object.schema = 'schema2';
 
+		// expected schema values for the dropdown that will be created
 		let expectedSchemaValues = [
 			{ displayName: 'schema2', name: 'schema2' }, // This should be the first database as it is active in the extension.
 			{ displayName: 'schema1', name: 'schema1' },
 			{ displayName: 'schema3', name: 'schema3' }
 		];
 
-		mockImportModel.object.schema = 'schema2';
+		// creating a mock connection
 		mockImportModel.object.server = {
 			providerName: 'MSSQL',
 			connectionId: 'testConnectionId',
 			options: {}
 		};
-		mockQueryProvider.setup(x => x.runQueryAndReturn(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(async () => { return schemaQueryResult });
+
+		// setting up mocks to return test objects created earlier
+		mockQueryProvider.setup(x => x.runQueryAndReturn(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(async () => { return schemaQueryResult; });
 		mockApiWrapper.setup(x => x.getProvider<azdata.QueryProvider>(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => { return mockQueryProvider.object; });
 
-		let importPage = new FileConfigPage(mockFlatFileWizard.object, TypeMoq.It.isAny(), mockImportModel.object, TypeMoq.It.isAny(), TypeMoq.It.isAny(), mockApiWrapper.object);
-		let actualSchemaValues = await importPage.getSchemaValues();
+		// creating a fileConfig Page and calling getSchema Value
+		let fileConfigPage = new FileConfigPage(mockFlatFileWizard.object, TypeMoq.It.isAny(), mockImportModel.object, TypeMoq.It.isAny(), TypeMoq.It.isAny(), mockApiWrapper.object);
+		let actualSchemaValues = await fileConfigPage.getSchemaValues();
 
+		// verifying if the correct values were returned by the getSchema method.
 		should(expectedSchemaValues).deepEqual(actualSchemaValues);
 	});
+
+	it('checking if all components are initialized properly', async function () {
+
+		// Opening the wizard and initializing the page as FileConfigPage
+		await new Promise(function (resolve) {
+			page.registerContent(async (view) => {
+				fileConfigPage = new FileConfigPage(mockFlatFileWizard.object, page, mockImportModel.object, view, TypeMoq.It.isAny(), mockApiWrapper.object);
+				pages.set(1, fileConfigPage);
+				await fileConfigPage.start().then(() => {
+					resolve();
+				});
+			});
+			wizard.generateScriptButton.hidden = true;
+
+			wizard.pages = [page];
+			wizard.open();
+		});
+
+		// checking if all the required components are correctly initialized
+		should.notEqual(fileConfigPage.serverDropdown, undefined);
```

Comment:
Make sure you add messages to these so that if one fails we know which one failed without needing to debug into it